content/docs/user-guide/checkpoints.md

```diff
@@ -0,0 +1,195 @@
+│ workspace     │ -        │    9 │ 0.9771 │
+│ live          │ 01:04 PM │    - │      - │
+│ │ ╓ exp-d3528 │ 01:21 PM │    9 │ 0.9771 │
+│ │ ╟ de0f0eb   │ 01:21 PM │    8 │  0.979 │
+│ │ ╟ 05e3cea   │ 01:21 PM │    7 │  0.974 │
+│ │ ╟ d0cd9ca   │ 01:20 PM │    6 │ 0.9708 │
+│ │ ╟ 0f495c4   │ 01:20 PM │    5 │ 0.9703 │
+│ │ ╟ 1131cba   │ 01:20 PM │    4 │ 0.9641 │
+│ │ ╟ 359869d   │ 01:20 PM │    3 │ 0.9521 │
+│ │ ╟ 9b0bf16   │ 01:20 PM │    2 │ 0.9271 │
+│ │ ╟ 39d1444   │ 01:19 PM │    1 │ 0.9157 │
+│ ├─╨ fcefafb   │ 01:19 PM │    0 │ 0.8572 │
+└───────────────┴──────────┴──────┴────────┘
+```
+
+Depending on the metrics you have in place, it can also show the accuracy of the
+model at each checkpoint.
+
+These experiments have IDs associated with each of the checkpoints so you have a
+reference to each epoch.
+
+## Starting from an existing checkpoint
+
+When you have checkpoints enabled in your `dvc.yaml` file, DVC will search for
+any `ini.txt` files related to that stage. If DVC finds any of these files, it
+will pick up the next training epoch based on the most recent checkpoint
+`ini.txt` file.
+
+If you want to start from a specific existing checkpoint, you'll need to run
+`dvc exp apply 123455` where `12345` is the checkpoint ID for specific
+checkpoint you want to start from. This creates a new experiment branch with a
+different set of checkpoints, starting with the one you specified.
+
+```dvc
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━┳━━━━━━━━┓
+┃ Experiment            ┃ Created  ┃ step ┃    acc ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━╇━━━━━━━━┩
+│ workspace             │ -        │    8 │ 0.9771 │
+│ live                  │ 01:04 PM │    - │      - │
+│ │ ╓ exp-6da78         │ 02:12 PM │    8 │ 0.9771 │
+│ │ ╟ 5209799           │ 02:12 PM │    7 │ 0.9693 │
+│ │ ╟ 2351030           │ 02:12 PM │    6 │ 0.9734 │
+│ │ ╟ 604156c (7fa8f60) │ 02:12 PM │    5 │ 0.9562 │
+│ │ ╓ exp-aba4c         │ 02:09 PM │    4 │ 0.9641 │
+│ │ ╟ 873e398           │ 02:09 PM │    3 │ 0.9521 │
+│ │ ╟ 7fa8f60           │ 02:09 PM │    2 │ 0.9271 │
+│ │ ╟ 345f462           │ 02:09 PM │    1 │ 0.9157 │
+│ ├─╨ ec28a11           │ 02:08 PM │    0 │ 0.8572 │
+│ │ ╓ exp-d3528         │ 01:21 PM │    9 │ 0.9771 │
+│ │ ╟ de0f0eb           │ 01:21 PM │    8 │  0.979 │
+│ │ ╟ 05e3cea           │ 01:21 PM │    7 │  0.974 │
+│ │ ╟ 9b0bf16           │ 01:20 PM │    2 │ 0.9271 │
+│ │ ╟ 39d1444           │ 01:19 PM │    1 │ 0.9157 │
+│ ├─╨ fcefafb           │ 01:19 PM │    0 │ 0.8572 │
+└───────────────────────┴──────────┴──────┴────────┘
+```
+
+### Resetting checkpoints
+
+When you need to completely retrain a model and you don't want to use any of the
+existing checkpoints, you can get rid of all of them with the `--reset` flag.
+Add this to the usual command for running DVC experiments and it looks like
+this: `dvc exp run --reset`.
+
+You can also reset checkpoints with the following command:
+`dvc exp apply && modify file && dvc exp run`.
```

Comment:
That's not clear 😄  What needs to be modified? Removing `checkpoint:true` from `dvc.yaml`? If so, it is a special case and probably should be in the basic tutorial.